components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/authz/handlers/util/ResponseTypeHandlerUtil.java

```diff
@@ -330,6 +331,18 @@ private static void addUserAttributesToCache(String accessToken, OAuthAuthzReqMe
                 authorizationGrantCacheEntry);
     }
 
+    private static AccessTokenDO getAccessTokenCheckingContext(String accessToken,
```

Comment:
Shall we change the method name something like "getAccessTokenDO"?

Reply:
Alright, we can change it